### PR TITLE
Use tabpane for JS libraries doc

### PR DESCRIPTION
**Pull request by cartermp**
As mentioned in https://github.com/open-telemetry/opentelemetry.io/issues/1772 let's try this out

### CODE REVIEW

**Comment by chalin** [comment]
We might have to update Docsy for this to work. I'll check when I'm next at my desk.

**Comment by cartermp** [comment]
Doesn't appear to need an upgrade: https://deploy-preview-1992--opentelemetry.netlify.app/docs/instrumentation/js/libraries/#node-autoinstrumentation-package

File: content/en/docs/instrumentation/js/libraries.md [hunk 1]
```diff
@@ -141,9 +139,9 @@ npm install --save @opentelemetry/instrumentation-http @opentelemetry/instrument
 
 And then register each instrumentation library:
 
-{{< ot-tabs TypeScript JavaScript >}}
+{{< tabpane >}}
```
Comment (chalin) [inline content/en/docs/instrumentation/js/libraries.md:142]:
```suggestion
{{< tabpane langEqualsHeader=true >}}
```

File: content/en/docs/instrumentation/js/libraries.md [hunk 2]
```diff
@@ -141,9 +139,9 @@ npm install --save @opentelemetry/instrumentation-http @opentelemetry/instrument
...
 
-{{< ot-tab >}}
+{{< tab header="TypeScript" lang="ts" >}}
```
Comment (chalin) [inline content/en/docs/instrumentation/js/libraries.md:144]:
```suggestion
{{< tab TypeScript >}}
```

File: content/en/docs/instrumentation/js/libraries.md [hunk 3]
```diff
@@ -181,9 +179,9 @@ const processor = new BatchSpanProcessor(exporter);
 provider.addSpanProcessor(processor);
 
 provider.register();
-{{< /ot-tab >}}
+{{< /tab >}}
 
-{{< ot-tab >}}
+{{< tab header="JavaScript" lang="js" >}}
```
Comment (chalin) [inline content/en/docs/instrumentation/js/libraries.md:184]:
```suggestion
{{< tab JavaScript >}}
```

**Review by chalin — APPROVED** [review]
See inline comments for a change that makes the `{{< tab >}}` opening tag more concise. Otherwise, LGTM!

File: content/en/docs/instrumentation/js/libraries.md [hunk 4]
```diff
@@ -44,9 +44,9 @@ npm install @opentelemetry/auto-instrumentations-node
 
 Then in your tracing initialization code, use `registerInstrumentations`:
 
-{{< ot-tabs TypeScript JavaScript >}}
+{{< tabpane >}}
 
-{{< ot-tab >}}
+{{< tab header="TypeScript" lang="ts" >}}
```
[the remark this answers is not in the document]
Comment (chalin) [reply content/en/docs/instrumentation/js/libraries.md:49]: Feel free to add quotes around the header text if that makes you feel more comfortable, but quotes are only required if there are spaces or special characters in the header text. (FYI)

**Pull request merged**